src/jit/codegenarm.cpp

```diff
@@ -1566,6 +1192,22 @@ void CodeGen::genCodeForCompare(GenTreeOp* tree)
 }
 
 //------------------------------------------------------------------------
+// genCodeForJcc: Produce code for a GT_JCC node.
+//
+// Arguments:
+//    tree - the node
+//
+void CodeGen::genCodeForJcc(GenTreeJumpCC* tree)
+{
+    assert(compiler->compCurBB->bbJumpKind == BBJ_COND);
+
+    CompareKind  compareKind = ((tree->gtFlags & GTF_UNSIGNED) != 0) ? CK_UNSIGNED : CK_SIGNED;
```

Comment:
When moving code around (e.g., between files), I tried not to change it at all. There are lots of places we could move to more "modern" style, like this, or using the new(-ish) OperIs() function.

I also didn't do the same refactoring on xarch, because I didn't want to touch it before the last 2.0  branch from master.